training/model_evaluation/src/task_factory/tabular/classification.py

```diff
@@ -27,7 +30,7 @@ def _alternate_predict(self, X_test, **kwargs):
         Returns:
             _type_: _description_
         """
-        device = kwargs.get("device", "cpu")
+        device = kwargs.get("device", -1)
```

Comment:
For Tabular Classifier we want to retain Default Compute to -1? How about other Classifiers (Text / Image)?

Reply:
"cpu" and "-1" are same. They both represent cpu in torch.